tests/components/Fees.spec.tsx

```diff
@@ -8,6 +8,7 @@ import { cfg } from "../config";
 
 describe("Fees component", () => {
     beforeAll(() => {
+        // @ts-ignore
```

Comment:
The type of `setConfig` is wrong. Don't just ts-ignore here. That defeats the purpose of typescript